plotly_resampler/figure_resampler/figure_resampler.py

```diff
@@ -0,0 +1,151 @@
+        default_n_shown_samples: int = 1000,
+        default_downsampler: AbstractSeriesAggregator = EfficientLTTB(),
+        resampled_trace_prefix_suffix: Tuple[str, str] = (
+            '<b style="color:sandybrown">[R]</b> ',
+            "",
+        ),
+        show_mean_aggregation_size: bool = True,
+        verbose: bool = False,
+    ):
+        assert isinstance(figure, go.Figure)
+        super().__init__(
+            figure,
+            convert_existing_traces,
+            default_n_shown_samples,
+            default_downsampler,
+            resampled_trace_prefix_suffix,
+            show_mean_aggregation_size,
+            verbose,
+        )
+
+        # The figureAggregator needs a dash app
+        self._app: JupyterDash | Dash | None = None
+        self._port: int | None = None
+        self._host: str | None = None
+
+    def show_dash(
+        self,
+        mode=None,
+        config: dict | None = None,
+        graph_properties: dict | None = None,
+        **kwargs,
+    ):
+        """Registers the :func:`update_graph` callback & show the figure in a dash app.
+
+        Parameters
+        ----------
+        mode: str, optional
+            Display mode. One of:\n
+              * ``"external"``: The URL of the app will be displayed in the notebook
+                output cell. Clicking this URL will open the app in the default
+                web browser.
+              * ``"inline"``: The app will be displayed inline in the notebook output
+                cell in an iframe.
+              * ``"jupyterlab"``: The app will be displayed in a dedicated tab in the
+                JupyterLab interface. Requires JupyterLab and the ``jupyterlab-dash``
+                extension.
+            By default None, which will result in the same behavior as ``"external"``.
+        config: dict, optional
+            The configuration options for displaying this figure, by default None.
+            This ``config`` parameter is the same as the dict that you would pass as
+            ``config`` argument to the `show` method.
+            See more https://plotly.com/python/configuration-options/
+        graph_properties: dict, optional
+            Dictionary of (keyword, value) for the properties that should be passed to
+            the dcc.Graph, by default None.
+            e.g.: {"style": {"width": "50%"}}
+            Note: "config" is not allowed as key in this dict, as there is a distinct
+            ``config`` parameter for this property in this method.
+            See more https://dash.plotly.com/dash-core-components/graph
+        **kwargs: dict
+            Additional app.run_server() kwargs. e.g.: port
+
+        """
+        graph_properties = {} if graph_properties is None else graph_properties
+        assert "config" not in graph_properties.keys()  # There is a param for config
+        # 1. Construct the Dash app layout
+        app = JupyterDash("local_app")  # TODO -> was jupyterdash
```

Comment:
What do you mean with your TODO?

Reply:
Aahn changed it temporarily to dash for some testing purposes, comment can be removed!!

Reply:
Okay will remove it! :broom: